core/resharding.go

```diff
@@ -0,0 +1,173 @@
+package core
+
+import (
+	"math/rand"
+	"sort"
+	"strconv"
+
+	"github.com/harmony-one/harmony/core/types"
+)
+
+const (
+	// InitialSeed is the initial random seed, a magic number to answer everything, remove later
+	InitialSeed int64 = 42
```

Comment:
can you use a random number here?

Reply:
Yes, we can use a random number here. Since this is only for testing purpose, so I just fix a number to see whether each run get the same state for each epoch